src/assets/sass/base.scss

```diff
@@ -48,20 +48,70 @@ h6 {
   background: transparent;
   border: 1px solid $color-danger;
   border-radius: 60px;
+  box-shadow: none;
   color: $color-danger;
   cursor: pointer;
   display: inline-block;
   font-size: 16px;
+  height: auto;
   outline: transparent;
-  padding: 15px 30px;
   transition: all 0.3s;
 
-  &:hover {
+  &:hover,
+  &:focus {
     background: $color-danger;
+    box-shadow: none;
     color: $color-white;
   }
 }
+
 .btn-lg {
   display: block;
-  width:100%;
-}
\ No newline at end of file
+  width: 100%;
```

Comment:
Nice job with your styling. I think it's tautologous to use `display: block` and `width: 100%` here. What do you think?